pages/learn/manage/device-statuses.md

```diff
@@ -11,36 +11,69 @@ Device statuses are displayed on the Devices page and the Device Summary page. A
 | **Inactive**                                 | The device has been [deactivated][deactivated] or has been [preregistered][preregistered] but has not yet connected to the {{ $names.cloud.lower }} API.                         |
 | **Frozen**                                   | The device has been frozen because it's outside the organization's allowance, or is in a paid [fleet type][fleet type] on a free tier organization.                              |
 
+
 ## Device Connectivty states
 
+
+
+The balena device uses two mechanisms to maintain communication with balenaCloud, namely a periodic HTTP API heartbeat and a VPN based tunnel called cloudLink. The state of both of these links can be seen in the device listing.
+
 ![Device connectivity indicators](/img/common/main_dashboard/device_status.webp)
 
-When the device's network connectivity is fully operational, the status displays as Online, which is comprised of healthy Heartbeat and VPN statuses. A device's status may include indicators of partial connectivity issues on the device, such as a firewall blocking VPN traffic or a device's inability to communicate with the {{ $names.cloud.lower }} API. These indicators are as follows:
- 
-<sup>1</sup> `Online (VPN Only)` indicates that the device is unable to communicate with the {{ $names.cloud.lower }} API. A device with a VPN Only status is not able to apply any new changes made such as deploying new releases, applying service configuration values, or switching to local mode. However, it is accessible via SSH or the web terminal. Performing an action such as rebooting or restarting containers might work, but most likely will not. This is because the device loses its Heartbeat if it's not communicating with the API, which is usually when the Supervisor on the device is not running or crashing. Since we have VPN access, we can SSH into the device and investigate further.
+<sup>1</sup> `Reduced Functionality` indicates that the device is experiencing limited connectivity. This can be due to either **missing recent Heartbeat API communications** or **lack of connection to cloudlink**. The boolean columns for these states are as follows:
+
+- **Cloudlink Available, but heartbeat not**: The device is unable to communicate with the {{ $names.cloud.lower }} API. A device with cloudlink only is not able to apply any new changes made such as deploying new releases, applying service configuration values, or switching to local mode. However, it is accessible via SSH or the web terminal. Performing an action such as rebooting or restarting containers might work, but most likely will not. This is because the device loses its Heartbeat if it's not communicating with the API, which is usually when the Supervisor on the device is not running or crashing. Since we have cloudlink tunnel access, we can SSH into the device and investigate further.
```

Comment:
found this to read with a lack of confidence - 'might work' is probably not something to put in documentation.

"we have cloudlink tunnel access..." might need rephrasing too. 

```suggestion
- **Cloudlink Available, but heartbeat not**: A missing heartbeat means that the device is unable to communicate with the {{ $names.cloud.lower }} API. A device in this state is not able to apply any new changes made such as deploying new releases, applying service configuration values, or switching to local mode, due to a reliance on API connectivity for this functionality. A missing heartbeat is symptomatic of a problem with the device connectivity or Supervisor, meaning actions such as rebooting or restarting containers are likely to be unavailable. Device debugging is possible via the terminal in this state due to the availability of cloudlink which allows an on-demand connection to the device to be made.
```

Reply:
Why is 'Available' capitalised in both bullet points?